MIDI Editor/talagan_OneSmallStep.lua

```diff
@@ -9,6 +9,11 @@
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Change input mode.lua             > talagan_OneSmallStep/actions/talagan_OneSmallStep Change input mode - KeyboardPress.lua
```

Comment:
MIDIUtils.lua is not listed in the included files:

```  [nomain] talagan_OneSmallStep/lib/*.lua
```

Reply:
Yeah, it's in `talagan_OneSmallStep/classes/lib` and I stupidly thought that

`[nomain] talagan_OneSmallStep/classes/*.lua`

was recursive. Happily you were vigilant !! Correcting this right away.

Reply:
Done. Thanks for your watchfulness !

Reply:
Thanks for the tip, I've accepted your suggestion. Bonne nuit ! :)